images/hub/Dockerfile

```diff
@@ -6,7 +6,6 @@ USER root
 
 RUN python3 -m pip install --no-cache git+https://github.com/berkeley-dsep-infra/canvasoauthenticator@09e7ed8
 RUN python3 -m pip install --no-cache jupyterhub-ltiauthenticator==1.5.1
-RUN python3 -m pip install --no-cache git+https://github.com/jupyterhub/kubespawner@main
```

Comment:
I think we will still need:
```
RUN python3 -m pip install --no-cache git+https://github.com/jupyterhub/kubespawner@6.1.0
```
Otherwise kubespawner will be downgraded to the version in our helm chart, which was broken.

Reply:
got it!